libs/product/driver/magento/src/transforms/spec-data/daff-composite-product.json

```diff
@@ -2,6 +2,7 @@
 	"type": "composite",
 	"id": "22",
 	"url": "/url.html",
+  "canonicalUrl": "/canonical-url.html",
```

Comment:
For correctness, give this a full URL with protocol and domain.

Reply:
changed